host.go

```diff
@@ -237,7 +237,11 @@ func (h *Host) AutoCompleteFunction(u *message.User) func(line string, pos int,
 			if completed == "/reply" {
 				replyTo := u.ReplyTo()
 				if replyTo != nil {
-					completed = "/msg " + replyTo.Name()
+					name := replyTo.Name()
+					_, found := h.GetUser(name)
+					if !found {
+						completed = "/msg " + name
+					}
```

Comment:
if not found, could you also reset the state of u.ReplyTo? I think `u.SetReplyTo(nil)` should do the trick.